kusto/ingest/internal/resources/resources.go

```diff
@@ -44,7 +44,7 @@ type URI struct {
 	sas                             url.Values
 }
 
-// Parse parses a string representing a Kutso resource URI.
+// parses a string representing a Kutso resource URI.
```

Comment:
You need the name of the function in the start of the doc (one of those go things)

so it should be

// Parse parses a string ...